R/write_validation_report.R

```diff
@@ -0,0 +1,94 @@
+#' Create validation report from template
+#' @param pkg_name name of package
+#' @param template what validation report template from {valtools} to use,
+#' passed to \code{usethis::use_template}
+#' @param open boolean whether to open the validation report, passed to \code{usetthis::use_template}
+#' @importFrom usethis use_template proj_set
+#' @importFrom tools file_ext
+#' @importFrom rlang with_interactive
+#' @export
+vt_use_report <- function(pkg_name = desc::desc_get_field("Package"),
+                          template = "validation_report.Rmd",
+                          open = FALSE){
+
+  tryCatch({
+    dir <- vt_find_config()
+  },
+  error = function(e){
+    abort(
+      paste0("Validation config file missing; instantiate via `vt_use_config()`."),
+      class = "vt.validation_config_missing"
+    )
+  })
+
+  val_leads <- get_val_leads(dir = dirname(dir))
```

Comment:
Now that I see what is going on here, you could probably just do `val_leads <- get_val_leads(dir = vt_path())` the same logic would hold, and I don't see `dir` being used elsewhere?

Reply:
how should we be retrieving the name of package being validated? this isn't currently being captured via validation.yml, but we won't always be able to access DESCRIPTION in the parent folders of `/validation/` e.g. validate as separate activity